src/plugins/console/public/lib/es/es.ts

```diff
@@ -39,17 +42,34 @@ export async function send({
   withProductOrigin = false,
   asResponse = false,
 }: SendProps) {
-  const options: HttpFetchOptions = {
+  const isKibanaApiRequest = path.includes(KIBANA_API_KEYWORD);
```

Comment:
Seeing that we need to check for kibana request in at least 2 places, could we maybe create a function to re-use and have the logic of checking the path in one place?